OpenWeather/App/Home/HomeViewModel.swift

```diff
@@ -23,30 +43,30 @@ final class HomeViewModel: ObservableObject, ErrorHandling {
     @Published var windSpeed: String = ""
 
     init(
-        weatherProviding: WeatherProviding,
-        locationProviding: LocationProviding,
+        capabilities: HomeCapable,
```

Comment:
This can either be a protocol or a concrete value. There could be a struct inside the ViewModel similar to Input and Content.